docs/deploy/howto-install-iot-edge.md

```diff
@@ -1,18 +1,26 @@
-# How to Install Azure IoT Edge for Industrial Scenarios
+# How to Setup a Local Gateway for Industrial Scenarios
 
-[Home](readme.md)
+The industrial assets (machines and systems) are connected to Azure through modules running on an [Azure IoT Edge](https://azure.microsoft.com/services/iot-edge/) industrial gateway.
 
-The industrial assets (machines and systems) are connected to Azure through modules running on an [Azure IoT Edge](https://azure.microsoft.com/services/iot-edge/) gateway. This article explains how to setup an IoT Edge Gateway for industrial scenarios.
+You can purchase industrial gateways compatible with IoT Edge. Please see our [Azure Device Catalog](https://catalog.azureiotsolutions.com/alldevices?filters={%223%22:[%222%22,%229%22],%2218%22:[%221%22]}) for a selection of industrial-grade gateways. Alternatively, you can setup a local VM.
 
-## IoT Edge Runtime
+## Automatic Industrial IoT Gateway Installation
 
-You can purchase preconfigured IoT Edge gateways, please see our Azure Device Catalog for a selection. Alternatively, you can install the IoT Edge runtime following the Azure IoT Edge [documentation](https://docs.microsoft.com/en-us/azure/iot-edge/). You can install the runtime on [Linux](https://docs.microsoft.com/en-us/azure/iot-edge/how-to-install-iot-edge-linux) or [Windows](https://docs.microsoft.com/en-us/azure/iot-edge/how-to-install-iot-edge-windows).
+Run the [Industrial IoT Gateway Installer](quickstart-gateway-installer.md) from your gateway to automatically install the IoT Edge Runtime and Industrial Modules.
 
-To support network scanning for equipment discovery, the Discovery Module should best run in Docker host networking mode. To enable host networking mode on Windows, follow the instructions [below](#Windows-Networking-Configuration).
+## Manual Industrial IoT Gateway Installation
```

Comment:
I think automatic/manual is misleading.   I would rather say - "Using the … Installer" vs. "Using the portal/az, etc.".

Reply:
I assume this means the installer already creates below "tags" and you remove the manifest installation?  This would be awesome...

Reply:
Correct. As requested. :-)